var/spack/repos/builtin/packages/dia/package.py

```diff
@@ -34,15 +34,18 @@ class Dia(Package):
 
     depends_on('intltool', type='build')
     depends_on('gtkplus@2.6.0:')
-    depends_on('cairo')
-    depends_on('libpng')
```

Comment:
You're sure these dependencies aren't needed? They might link to a different executable or library. I'm not familiar with dia though.